crates/ef-testing/src/models/case.rs

```diff
@@ -122,7 +126,7 @@ impl BlockchainTestCase {
         let block = test
             .blocks
             .first()
-            .ok_or(RunnerError::Other("test has no blocks".to_string()))?
+            .ok_or_else(|| RunnerError::Other("test has no blocks".to_string()))?
```

Comment:
this is clippy saying we should use ok_or_else so that the function within the brackets is lazily evaluated, 
if we use ok_or, it's run no matter what, 

so the takeaway is that you should use ok or else when using functions within the error case